test/test_norm_hypercube.txt

```diff
@@ -0,0 +1,1001 @@
+    9.9995
```

Comment:
if this is to test correctness of the code would it be simpler to have a test in much smaller dimension? The correctness is not affected by the dimension right?